src/app/export/export.component.spec.ts

```diff
@@ -12,6 +12,19 @@ import { MockExportCSVService, MockExportPDFService } from '../_mocked';
 import { ExportType, FmtTableData } from '../_models';
 import { ExportCSVService } from '../_services';
 
+window['URL'] = {
+  createObjectURL: (_) => {
+    return '';
+  },
+  revokeObjectURL: (_) => {}
+} as any;
+
+Object.assign(navigator, {
+  clipboard: {
+    writeText: () => {}
+  }
+});
```

Comment:
This seems a bit hackish... and I have seen it being introduced in other places. I suspect this is a way to disable some calls made from within the tested objects. Is there no more elegant way to handle this? Perhaps overriding the relevant functionality in the object to be tested with an empty function?